dev-db/sqlitebrowser/metadata.xml

```diff
@@ -1,15 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+
 <pkgmetadata>
-  <!-- maintainer-needed -->
-  <longdescription lang="en">
-SQLite Database Browser is a freeware, public domain, open source visual tool
-used to create, design and edit database files compatible with SQLite. It is
-meant to be used for users and developers that want to create databases, edit
-and search data using a familiar spreadsheet-like interface, without the need to
-learn complicated SQL commands.
-</longdescription>
-  <upstream>
-    <remote-id type="github">sqlitebrowser/sqlitebrowser</remote-id>
-  </upstream>
+	<maintainer type="person">
+		<email>beatussum@protonmail.com</email>
+		<name>Mattéo Rossillol‑‑Laruelle</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		DB Browser for SQLite (DB4S) is a freeware, public domain, open source
+		visual tool used to create, design and edit database files compatible
+		with SQLite. It is meant to be used for users and developers that want
+		to create databases, edit and search data using a familiar
+		spreadsheet-like interface, without the need to learn complicated SQL
+		commands.
+	</longdescription>
+	<longdescription lang="fr">
+		DB Browser for SQLite (DB4S) est un outil graphique, gratuit et open
+		source permettant de créer, concevoir et éditer toute base de données
+		compatible avec SQLite. Celui-ci est à déstination des utilisateurs et
+		des développeurs qui veulent créer des base de données et interragir
+		avec ces dernières, tout en utilisant une interface semblable à celle
+		d'un tableur, et, de ce fait, ne pas être contraint à apprendre des
+		commandes SQL complexes.
+	</longdescription>
+	<upstream>
+		<maintainer status="active">
+			<name>DB Browser for SQLite (DB4S) project</name>
+		</maintainer>
```

Comment:
I feel like this doesn't add any value. It's like saying "sqlitebrowser's maintainer is: sqlitebrowser's maintainers".